configuration.adoc

```diff
@@ -1701,7 +1701,14 @@ zmcontrol restart
 S/MIME is a standard to send secure email messages. S/MIME messages use
 digital signature to authenticate and encrypt messages.
 
-=== Prerequisites
+Currently, there are two versions of S/MIME feature
```

Comment:
Perhaps:
Currently, there are two different methods for providing the S/MIME feature